build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/AcceptAndroidSdkLicenses.cs

```diff
@@ -16,7 +16,11 @@ public override bool Execute ()
 			var licdir = Path.Combine (Path.Combine (AndroidSdkDirectory, "licenses"));
 			Directory.CreateDirectory (licdir);
 
-			var psi = new ProcessStartInfo (Path.Combine (AndroidSdkDirectory, "tools", "bin", "sdkmanager"), "--licenses") { UseShellExecute = false, RedirectStandardInput = true };
+			var path = Path.Combine (AndroidSdkDirectory, "tools", "bin", "sdkmanager");
```

Comment:
This should use `Which.GetProgramLocation()`, so that `%PATHEXT%` is used. This way, if `sdkmanager` becomes `sdkmanager.cmd` in the future, we'll do the sane thing.